test/benchmarks/benchmark_helper.rb

```diff
@@ -0,0 +1,11 @@
+module BenchmarkHelper
```

Comment:
Any particular reason we don't want to use the standard benchmark library? http://ruby-doc.org/stdlib-2.3.0/libdoc/benchmark/rdoc/Benchmark.html

Reply:
I was thinking in terms of iterations per second, which isn't part of the standard ruby library. I'm up to discuss this some more and swapping out measuring IPS for the standard benchmark if need be.

Reply:
^ Disregard.

Reply:
Ah, no worries. I think this is good! Thanks again for throwing this together so quickly.